app/src/main/java/app/revanced/manager/ui/destination/Destination.kt

```diff
@@ -11,16 +11,16 @@ import kotlinx.parcelize.RawValue
 sealed interface Destination : Parcelable {
 
     @Parcelize
-    object Dashboard : Destination
+    data object Dashboard : Destination
 
     @Parcelize
     data class InstalledApplicationInfo(val installedApp: InstalledApp) : Destination
 
     @Parcelize
-    object AppSelector : Destination
+    data object AppSelector : Destination
 
     @Parcelize
-    object Settings : Destination
+    data class Settings(val startDestination: SettingsDestination? = null) : Destination
```

Comment:
Its better and more readable to set a non-nullable default instead of replacing nulls with a default value.